dex/farm/src/lib.rs

```diff
@@ -613,4 +613,207 @@ pub trait Farm:
             &context.get_input_attributes().unwrap().compounded_reward,
         )
     }
+
+    //This function is neccessary in old farms.
+    #[payable("*")]
+    #[endpoint(migrateToNewFarm)]
+    fn migrate_to_new_farm(
```

Comment:
Let's disable all endpoints when migration starts. ExitFarm and migrateToNewFarm should be enabled only.

Reply:
done